src/components/SharedComponents/BottomSheet.js

```diff
@@ -3,17 +3,17 @@
 import React from "react";
 import { View } from "react-native";
 import type { Node } from "react";
-import { viewStyles } from "../../styles/sharedComponents/bottomModal";
+import { viewStyles } from "../../styles/sharedComponents/bottomSheet";
 
 type Props = {
   children: any,
   height?: number
 }
 
-const BottomModal = ( { children, height }: Props ): Node => (
-  <View style={[viewStyles.bottomModal, { height }]}>
+const BottomSheet = ( { children, height }: Props ): Node => (
+  <View style={[viewStyles.bottomSheet, { height }]}>
     {children}
   </View>
 );
 
-export default BottomModal;
+export default BottomSheet;
```

Comment:
You might consider re-doing this with React Native Bottom Sheet, a 3rd-party dep we just added as a part of the Add ID component. At a mininum we could have consistent styling, but if the bottom sheet for the logged out state ever needs more interactivity that might make it easier to support.